woocommerce/woocommerce-ios
# Move Payment Section to Right After Products and Refunded Products

Closes #2282

This changes the order of the sections in the Order Details so that the Payment will be shown right after Products and Refunded Products. The Customer section is shown after the Payment section.

## Design

Before | After 
--------|-------
![before](https://user-images.githubusercontent.com/198826/82070029-69f04600-9691-11ea-8c04-a24416dce232.gif)        |       ![after](https://user-images.githubusercontent.com/198826/82070041-6c52a000-9691-11ea-8fe5-e2cb0a86a0b9.gif)

## Testing

1. Navigate to an order that has a refunded product. 
2. Confirm that the Payment section is shown after the Refunded Products section.
3. Navigate to an order that has no refunded product. 
4. Confirm that the Payment section is shown after the Products section. 

## Reviewing 

Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
